### ISSUE TITLE
Handle input with wrong dimensions more gracefully

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/brainreg#128]
We should provide a better message to users passing e.g. a stack of RGB images to brainreg, see e.g. https://forum.image.sc/t/brainreg-error-when-checking-orientation-value-error-axes-dont-match-array/83794/3

**Comment by adamltyson** [comment]
I guess we shoud add two checks:
- If dimensionality >3 OR
- dimensionality =3 and one dimension is length 2-N(6?)

then raise an error explaining what the input data should be.

I guess we could also provide functionality within the napari plugin to process one channel of a multichannel image.

**Comment by Aditya-Sinha1** [comment]
Hey I would like to work on this issue

There seems to be a check in load_img_stack function in brainglobe_utils/IO/image/load.py that checks dimensions of the read tiff file, but it only raises 2d tiff image exception and it might also allow 2d multi channel tiff images

another check exists in the same file in the function _verify_tiff_contains_single_3D_stack but it just registers a log warning

Sorry if I am missing something

**Comment by adamltyson** [comment]
Yep I thik there are a few existing checks, but maybe not one for this specific issue.
